src/sections/Community/Handbook/contributing.js

```diff
@@ -4,85 +4,214 @@ import { HandbookWrapper } from "./Handbook.style";
 import TOC from "../../../components/handbook-navigation/index";
 import Code from "../../../components/CodeBlock";
 import Signoff from "../../../../.github/assets/images/git-signoff-vscode.png";
-import github from "../../../assets/images/socialIcons/github.svg";
 import TocPagination from "../../../components/handbook-navigation/TocPagination";
+import { Link } from "gatsby";
+import IntraPage from "../../../components/handbook-navigation/intra-page";
 
-const contributingGuide= () => {
+const contents = [
+  {
```

Comment:
`Contribution` page has incorrect intra-page navigation.

Reply:
@debo19 yeah i do not know what the navigation titles should be on this page.

Reply:
- Clone your fork
- Checkout a new branch
- Commit your changes
- Push changes to Github
- Sign-off commits